src/views/partials/board-settings.handlebars

```diff
@@ -0,0 +1,57 @@
+
+<!-- Css for dropdown menu on w3schools -->
+<div class="dropdown">
+    <button class="dropbtn">Settings</button>
+    <div class="dropdown-content">
+      <a href="#" class="btn btn-link">Board Settings</a>
+      <a href="#" class="btn btn-link">Invite</a>
+    </div>
+</div> 
+
+
+<div id="boardSettingsModal" class="modal">
+    <!-- Exit button (needs css) -->
+    <label for="Close Board Settings" class="close-btn">X</label>
+    
+    <!-- Form for changing the board settings -->
+    <form action="/board" method="POST">
+        <div class="boardNameDiv">
+            <label for="Board Name">Name: </label>
+            <input id="boardName" type="text" placeholder="board name" value="{{board.boardName}}"/>
```

Comment:
you need a name attribute, so the backend knows what the data is called (DO THIS FOR ALL)

Since the type="text", value and placeholder have the same functionality (as far as I can tell from online). So drop the placeholder (unless you know otherwise).

Reply:
@mpolimen 
In this scenario, "board name" would show when the input is empty, but the point is that the input box starts with the value of board.boardName, but will still have the placeholder showing when the input is empty. I think it's fine

Reply:
Okay, that makes sense 👍